infrastructure/metadata/infrastructure/server/webtransport-h3.sub.any.js.ini

```diff
@@ -0,0 +1,29 @@
+[webtransport-h3.sub.any.html]
+  [WebTransport server should be running and should handle a bidirectional stream]
+    expected:
+      if product == "safari": [FAIL, ERROR]
```

Comment:
From the previous logs, the only ERROR I can see for Safari is for webtransport-h3.sub.any.sharedworker.html, which is because shared workers aren't supported. All the other variants of the test should be failing in the usual manner. Can you simplify this and the rest except in `[webtransport-h3.sub.any.sharedworker.html]` to just "FAIL"?

Reply:
Ah I see. Done.